### DISCUSSION TITLE
Item details in price checking window

### DISCUSSION THREAD

**Discussion by freezerain** [SnosMe/awakened-poe-trade#1280] [Ideas]
Sometimes when I want to check what mods have item I competing with, I am clicking "trade" button to open official trade website. I think it will be nice to have this info inside price checking window, for example below trade button there are enough space to put a "Details" buttons column, which on hover can show item mods and rols. Also this will unload some api calls.

![image](https://github.com/SnosMe/awakened-poe-trade/assets/35431136/d1ae7390-a5d8-472a-a10b-e8eef20b1c35)

**Comment by SnosMe** [comment]
It's nice to have, but it's a relativly big feature and I already have a lot of things to maintain.
